pretyping/arguments_renaming.ml

```diff
@@ -69,19 +69,25 @@ let rename_arguments local r names =
 
 let arguments_names r = GlobRef.Map.find r !name_table
 
-let rec rename_prod c = function 
+let get_rename old_name rename_item =
```

Comment:
I think you should introduce `Name.default`, if it does not already exists.

Reply:
The type does not match the one of, say, `Option.default` since it is
```val default : default:Name.t -> Name.t -> Name.t```
I could not find a name for such a combinator.. help is welcome

Reply:
I've renamed the function `name_override old_name override` after a discussion with Cyril.
Wrt moving it to `kernel/names.ml` I did not do it because It is not a standard combinator AFAICT. (`default` is and hence its place would have been there, but given it is not `default`...).